pkg/provider/vsphere/plugin.go

```diff
@@ -48,7 +60,8 @@ func NewVSphereInstancePlugin(vc *vCenter, ignoreOnDestroy bool) instance.Plugin
 	internals, err := vCenterConnect(vc)
 	if err != nil {
 		// Exit with an error if we can't connect to vCenter (no point continuing)
-		log.Fatalf("%v\n", err)
+		log.Crit("Error connecting to vCenter", "response", err.Error())
+		os.Exit(-1)
```

Comment:
I think it's better to let this error bubble up the stack, rather than just exiting here.  This will make it possible for the upper layer to do things in the future like retries without you having to do it yourself here.

Reply:
Wasn’t sure of the code path.  Didn’t seem to make sense replying with a plugin struct when the config was incorrect. I was also wondering why there wasn’t a fatal option as part of the logging. I need to re-consider the auth strategy for this part of the plugin moving forward.